novuRajan/gannt-chart
# Relationship UI( Tasks dependency line)

This pull request resolves the issue related to the rendering of relationship lines between dependent tasks in the Gantt chart, enhancing visual clarity. The changes made ensure that the lines and arrowheads are correctly positioned and visually connected to their dependent tasks.

**Changes Made:**

Added the new function to draw the relationship line with arrowheads pointing to dependent tasks.


**Testing instructions**

Open the Gantt chart with tasks that have dependencies, currently is for relationship between parent tasks.
Confirm that the relationship lines leading to the arrowheads are now rendered with enhanced clarity.
